lib/identity_config.rb

```diff
@@ -314,7 +314,6 @@ def self.build_store(config_map)
     config.add(:max_piv_cac_per_account, type: :integer)
     config.add(:min_password_score, type: :integer)
     config.add(:minimum_wait_before_another_usps_letter_in_hours, type: :integer)
-    config.add(:multi_region_kms_migration_jobs_enabled, type: :boolean)
     config.add(:multi_region_kms_migration_jobs_profile_count, type: :integer)
     config.add(:multi_region_kms_migration_jobs_profile_timeout, type: :integer)
     config.add(:multi_region_kms_migration_jobs_user_count, type: :integer)
```

Comment:
Should we remove the  other configurations?

Reply:
good catch; just cleaned up